### PR TITLE
refac: Rename domain tests to subsystem tests

### PR DESCRIPTION
**Pull request by BjarkeMeier**
OBS: Requires a corresponding CD change to be merged first.

### CODE REVIEW

File: .github/actions/dotnet-prepare-outputs/action.yml
Comment (dstenroejl) [inline .github/actions/dotnet-prepare-outputs/action.yml:1]: The file `cd.yml` should be updated:

The `event-type: wholesale-deployment-request-domain` should be `event-type: wholesale-deployment-request-subsystem`

The start of line containing:
client-payload: '{"domain_pr":
should be
client-payload: '{"subsystem_pr":

**Review by dstenroejl — COMMENTED** [review]
Commented

File: .github/actions/dotnet-prepare-outputs/action.yml
Comment (BjarkeMeier) [reply .github/actions/dotnet-prepare-outputs/action.yml:1]: According to the PR title, the scope is solely "domain test" and not everything related to the term "domain".
I'd like to keep the PR small and focused. And leave the rest for other PRs.

File: .github/actions/dotnet-prepare-outputs/action.yml [hunk 1]
```diff
@@ -52,12 +52,12 @@ runs:
           --configuration Release \
           --output '${{ inputs.output_path }}\databasemigration'
 
-    - name: Publish Domain Tests
+    - name: Publish Subsystem Tests
```
[the remark this answers is not in the document]
Comment (BjarkeMeier) [reply .github/actions/dotnet-prepare-outputs/action.yml:55]: According to the PR title, the scope is solely "domain test" and not everything related to the term "domain".
I'd like to keep the PR small and focused. And leave the rest for other PRs.

File: source/dotnet/subsystem-tests/Clients/Wholesale/V3/swagger.json
[the remark this answers is not in the document]
Comment (BjarkeMeier) [reply source/dotnet/subsystem-tests/Clients/Wholesale/V3/swagger.json:1]: I'm not sure where these values originates. But this is generated code so it's just a consequence of the code being regenerated/updated in this PR.

**Review by dstenroejl — APPROVED** [review]

**Review by Renetnielsen — APPROVED** [review]

**Pull request merged**